portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -418,4 +534,184 @@ definition {
+				locator1 = "Icon#ANY_N");
+		}
+
+		task ("When the user hovers the tooltip icon") {
+			MouseOver(
+				key_indexNumber = 1,
+				key_text = "question-circle-full",
+				locator1 = "Icon#ANY_N");
+		}
+
+		task ("Then the message “This key allows the action to be performed when the permission is associated with it” is displayed.") {
+			AssertElementPresent(
+				key_title = "",
+				locator1 = "DataSet#FILTER_TOOLTIP",
+				value1 = "This key is used to display the action if the user has the permission for it by checking if the key is present in the data item's actions array.");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the Icon field is not mandatory and shows the correct icon"
+	@priority = 4
+	test IconFieldIsNotMandatory {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then the New Item Action modal is displayed") {
+			AssertElementPresent(
+				key_title = "New Item Action",
+				locator1 = "Header#H2_TITLE");
+		}
+
+		task ("Then the Icon field is not mandatory") {
+			AssertElementNotPresent(
+				key_fieldName = "Icon",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
+
+		task ("When the user clicks on the Icon field") {
+			Click(
+				key_fieldLabel = "Icon",
+				locator1 = "Select#GENERIC_SELECT_FIELD");
+		}
+
+		task ("Then a dropdown list all the icons in clay with the name and the corresponding icon") {
+			Select(
+				key_fieldLabel = "Icon",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "accessibility");
+
+			AssertElementPresent(
+				key_text = "pt-br",
+				locator1 = "Icon#ANY");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the Confirmation Message field is not mandatory and is localized"
+	@priority = 4
+	test LocalizedConfirmationMessageField {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And selects the Link option") {
+			Select(
+				key_fieldLabel = "Type",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "Link");
+		}
+
+		task ("Then the Confirmation Message field is present") {
+			AssertElementPresent(
+				key_placeholder = "Add a message here.",
+				locator1 = "FrontendDataSet#PLACEHOLDER");
+		}
+	}
+
+	@description = "LPS-192149. Verify if the Label field is mandatory and localized in Action"
+	@priority = 5
+	test MandatoryAndLocalizedLabelField {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then the New Item Action modal is displayed ") {
+			AssertElementPresent(
+				key_title = "New Item Action",
+				locator1 = "Header#H2_TITLE");
+		}
+
+		task ("And the Label field is mandatory") {
+			AssertElementPresent(
+				key_fieldName = "Label",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
+
+		task ("And Label field is localized ") {
+			AssertElementPresent(
+				key_field = "Label",
+				key_localizedLanguage = "en_US",
+				locator1 = "DataSet#LOCALIZED_FIELD");
+		}
+	}
```

Comment:
Just checking the mandatory field is not enough, we should try to save and check that the action cannot be saved without filling the field